src/test/java/uk/ac/ebi/eva/contigalias/service/ChromosomeServiceIntegrationTest.java

```diff
@@ -65,17 +66,37 @@ void getChromosomeByRefseq() {
         assertChromosomePageIdenticalToEntity(page);
     }
 
+    @Test
+    void putChromosomeChecksumsByAccession() {
+        service.putChromosomeChecksumsByAccession(entity.getGenbank(),entity.getMd5checksum(), entity.getTrunc512checksum());
```

Comment:
if you take out this line, the test passes. This means that the test is not testing anything. 

Create an entity without checksums, save it, check that the checksums are missing, and then add the checksums, and then check they are not missing anymore.